c7n/resources/route53.py

```diff
@@ -59,15 +59,16 @@ def process_tags(resources):
                 k = k.split("/")[-1]
             resource_map[k] = r
 
-        results = retry(
-            client.list_tags_for_resources,
-            ResourceType=model.type,
-            ResourceIds=list(resource_map.keys()))
-
-        for resource_tag_set in results['ResourceTagSets']:
-            if ('ResourceId' in resource_tag_set and
-                    resource_tag_set['ResourceId'] in resource_map):
-                resource_map[resource_tag_set['ResourceId']]['Tags'] = resource_tag_set['Tags']
+        for r in list(resource_map.keys()):
```

Comment:
Rather than making an API call for each resource id, it would be better to chunk the list of keys into chunks of 10, a util function is available in c7n.utils to do that: https://github.com/capitalone/cloud-custodian/blob/master/c7n/utils.py#L201